crates/pagecache/src/config.rs

```diff
@@ -337,8 +337,8 @@ impl ConfigBuilder {
                         return Err(Error::Io(std::io::Error::new(
                                     std::io::ErrorKind::Other,
                                     format!(
-                                    "could not acquire appropriate file lock on {:?}",
-                                    path
+                                    "could not acquire appropriate file lock on {:?}: {:?}",
+                                    path, lock_res.unwrap_err()
```

Comment:
Would be better is you replace the `if lock_res.is_err()` by an `if let Err(e) = lock_res` and remove the `unwrap_err()`. :)

Reply:
Totally agree!

Reply:
Done!